bugbot/user_activity.py

```diff
@@ -77,6 +81,17 @@ def __init__(
         self.activity_limit_ts = lmdutils.get_date_ymd(self.activity_limit).timestamp()
         self.seen_limit = lmdutils.get_date(reference_date, self.absent_weeks_count * 7)
 
+        self.new_user_activity_limit = lmdutils.get_date(
+            reference_date, self.new_user_weeks_count
+        )
+        self.new_user_activity_limit_ts = lmdutils.get_date_ymd(
+            self.new_user_activity_limit
+        ).timestamp()
+        self.new_user_seen_limit = lmdutils.get_date(
+            reference_date, self.new_user_weeks_count * 7
+        )
+        self.new_user_limit = self.seen_limit = lmdutils.get_date(reference_date, 61)
```

Comment:
Why are you overwriting `self.seen_limit`?

Reply:
Thank you for pointing this out! I must have forgotten to remove `self.seen_limit` when copying the line over for `self.new_user_limit`. It has now been deleted here: 596dd0e8fb55cbd51ad19f8018f3fb9632aa7d2d